tests/etl/test_transformers.py

```
@@ -0,0 +1,101 @@
+from hypothesis import given, strategies as st
+
+
+@given(
+    st.lists(
+        st.text(
+            min_size=1,
+            alphabet="abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789 ()%+",
+        ),
+        min_size=1,
+        unique=True,
+    )
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)